doc/uftrace-live.md

```diff
@@ -322,6 +322,46 @@ Each field has following meaning:
 The default value is 'duration,tid'.  If given field name starts with "+", then it'll be appended to the default fields.  So "-f +time" is as same as "-f duration,tid,time".  And it also accepts a special field name of 'none' which disables the field display and shows function output only.
 
 
+DYNAMIC TRACING
+===============
+The uftrace tool supports dynamic function tracing which can be enabled and disabled at runtime on x86_64.  Before recording functions, you can build the target program with `-pg`, then it might have a tiny performance impact because all funtions call `mcount()`.  So you can partially enable given functions at load time with the `-P`/`--patch` option, after building the target program with not only `-pg` but also `-mfentry -mnop-mcount` to add 5 byte nop at function prologue.
+
+The following example shows a error message when normally running uftrace with the excutable built with `-pg -mfentry -mnop-mcount`. Because 'mcount' symbol can't be found.
+
+    $ gcc -o abc -pg -mfentry -mnop-mcount tests/s-abc.c
+    $ uftrace abc
+    uftrace: /home/namhyung/project/uftrace/cmd-record.c:1305:check_binary
+      ERROR: Can't find 'mcount' symbol in the 'abc'.
+             It seems not to be compiled with -pg or -finstrument-functions flag
+             which generates traceable code.  Please check your binary file.
+
+But when the `-P a` patch option is used, and then only it can dynamically trace `a()`.
+
+    $ uftrace -P a abc
+    # DURATION    TID     FUNCTION
+       1.685 us [20000] | __monstartup();
+       0.845 us [20000] | __cxa_atexit();
+                [20000] | a() {
+       0.615 us [20000] |   getpid();
+       1.380 us [20000] | } /* a */
+
+In addition, you can enable all functions at load time using '.' as a pattern with `P` option.
```

Comment:
Wouldn't it better adding a explanation that the '.' matches to any character in a regex pattern.